src/test/tcp/test_tcp.c

```diff
@@ -20,12 +20,19 @@
 #include <sys/uio.h>
 #include <time.h>
 #include <unistd.h>
+#include <glib.h>
```

Comment:
It doesn't look like you're actually using any glib functionality. If that's the case better to leave off the dependency for now.